.github/workflows/front-tests.yml

```
@@ -45,3 +45,10 @@ jobs:
     - name: Run Flutter tests
       run: flutter test
       working-directory: junqo_front
+
+  mark-tests-passed-if-no-changes:
+    if: needs.changes-check.outputs.any_changed != 'true'
```

actionlint

property "changes-check" is not defined in object type {}

(expression)